echo2.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "common/common/assert.h"
 
+namespace Lyft {
```

Comment:
Do we want to commit to Lyft as the namespace if one day this becomes CNCF? Maybe it should be Envoy? @mattklein123